packages/apps/spaces/app/src/components/EmailMultiCreatableSelect/MultiValueWithActionMenu.tsx

```diff
@@ -220,18 +219,7 @@ export const MultiValueWithActionMenu: FC<MultiValueWithActionMenuProps> = ({
         </MenuItem>
         {rest?.data?.value ? (
           <MenuItem
-            display='flex'
-            borderRadius='md'
-            border='1px solid'
-            borderColor='transparent'
-            _hover={{
-              bg: 'gray.50',
-              borderColor: 'gray.100',
-            }}
-            _focus={{
-              borderColor: 'gray.200',
-            }}
-            justifyContent='space-between'
+            className=' flex justify-between items-center rounded-md border border-transparent hover:bg-gray-50 hover:border-gray-100 focus:border-gray-200'
```

Comment:
I've recently implemented the Select again following the guides from figma. Why do we need to restyle them entirely? @steynviljoen67 regarding what we were discussing yesterday about consistency across the components we use.

Reply:
this is not migrated email select component with validation and correction of old values, I won't manage to move this whole functionality to new select today and after that I'm off - I propose to create separate issue for it

Reply:
But I see tw code in this and new components.